pom.xml

```diff
@@ -112,19 +112,19 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.3.2</version>
+            <version>2.13.5</version>
```

Comment:
Please use this version for all jackson dependencies. Probably worth to introduce a property for it.